test/powershell/Host/TabCompletion/BugFix.Tests.ps1

```diff
@@ -21,4 +21,11 @@ Describe "Tab completion bug fix" -Tags "CI" {
         $result.CompletionMatches[0].CompletionText | Should Be "-and"
         $result.CompletionMatches[1].CompletionText | Should Be "-as"
     }
+    It "Issue#2295 - '[pscu<tab>' should expand to [pscustomobject]" {
+        $result = TabExpansion2 -inputScript "[pscu" -cursorColumn "[pscu".Length
+        $result | Should Not BeNullOrEmpty
+        $result.CompletionMatches.Count | Should Be 2
+        $result.CompletionMatches[0].CompletionText | Should Be "pscustomobject"
+        $result.CompletionMatches[1].CompletionText | Should Be "psobject"
```

Comment:
I would argue about correctness of this test.
It's a current behavior, but it's not the right one.
Maybe just drop the check for count and check for [1] == `psobject` and leave a comment explaining what's going on?